ajax/solution.php

```diff
@@ -102,5 +102,18 @@
     $template->fields['content'] = RichText::getSafeHtml($content);
 }
 
+//load solutiontype name (use to create OPTION dom)
+//need when template is used and when GLPI preselcted type if defined
+if ($template->fields['solutiontypes_id']) {
```

Comment:
If the solution type is not available in the item entity scope, we should unset it.

Reply:
done